plat/xen/Config.uk

```diff
@@ -75,6 +75,10 @@ config XEN_GNTTAB
 	help
 		Support grant table operations
 
+config UKPLAT_ALLOW_GIC
```

Comment:
Isn't this a duplicate of [this](https://github.com/unikraft/unikraft/pull/726/commits/20c8dc8b96350c610a47a8563c5c8072b6f989a1#diff-fef3f72a280b4cfd00934570995e0ee66baca83b488d2ac2bc4ad16c0fbded0fR157)?

Reply:
fixed